staging/src/k8s.io/apiserver/pkg/server/hooks.go

```diff
@@ -196,7 +196,7 @@ func runPostStartHook(name string, entry postStartHookEntry, context PostStartHo
 	var err error
 	func() {
 		// don't let the hook *accidentally* panic and kill the server
-		defer utilruntime.HandleCrash()
+		defer utilruntime.HandleCrashWithContext(context)
```

Comment:
It's worth noting that `HandleCrash` still crashes by default. The comment seems wrong or out-dated.

Might be worth removing while touching this line.